test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/Metadata/DefaultBindingMetadataProviderTest.cs

```diff
@@ -514,6 +515,16 @@ public void GetBindingDetails_BindingBehaviorLeftAlone_ForAttributeOnPropertyTyp
             Assert.Equal(initialValue, context.BindingMetadata.IsBindingRequired);
         }
 
+        private static ModelBindingMessages CreateBindingMessages()
+        {
+            return new ModelBindingMessages
+            {
+                MissingBindRequiredValueResource = Resources.FormatModelBinding_MissingBindRequiredMember,
+                MissingKeyOrValueResource = Resources.FormatKeyValuePair_BothKeyAndValueMustBePresent,
+                ValueInvalid_MustNotBeNullResource = Resources.FormatModelBinding_NullValueNotValid,
+            };
+        }
```

Comment:
Should we put this in testcommon somewhere? Maybe expose off of `TestMetadataProvider`?